common/src/main/java/com/wynntils/features/inventory/CustomBankQuickJumpsFeature.java

```diff
@@ -258,15 +262,15 @@ protected void onConfigUpdate(Config<?> unknownConfig) {
         int maxValue;
 
         switch (fieldName) {
-            case "bankDestinations" -> {
-                containerType = SearchableContainerType.BANK;
-                maxValue = MAX_BANK_PAGES;
+            case "accountBankDestinations" -> {
+                containerType = SearchableContainerType.ACCOUNT_BANK;
+                maxValue = MAX_ACCOUNT_BANK_PAGES;
             }
             case "blockBankDestinations" -> {
                 containerType = SearchableContainerType.BLOCK_BANK;
                 maxValue = MAX_BLOCK_BANK_PAGES;
             }
-            case "bookshelfDestinations", "miscBucketDestinations" -> {
+            case "bookshelfDestinations", "miscBucketDestinations", "characterBankDestinations" -> {
                 containerType = SearchableContainerType.BOOKSHELF;
                 maxValue = MAX_HOUSING_CONTAINER_PAGES;
```

Comment:
Maybe the constant should be renamed now that it for more than housing?

Reply:
`MAX_DEFAULT_PAGES` as `HOUSING_DEFAULT_DESTINATIONS` was changed to just `DEFAULT_DESTINATIONS`? Can either add it to my next pr of if someone else wants to include it in theirs

Reply:
Yes,`MAX_DEFAULT_PAGES` sounds good. Add it to your next PR. Thanks!